include/read/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+# Создаем папку сборки для OpenCV
+file(MAKE_DIRECTORY "${CMAKE_SOURCE_DIR}/3rdparty/opencv/opencv/build")
+# Вызываем команды для сборки OpenCV через execute_process
+execute_process(
+    COMMAND ${CMAKE_COMMAND} -S "${CMAKE_SOURCE_DIR}/3rdparty/opencv/opencv" -B "${CMAKE_SOURCE_DIR}/3rdparty/opencv/opencv/build"
+    WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}/3rdparty/opencv/opencv/build"
+)
+execute_process(
+    COMMAND ${CMAKE_COMMAND} --build "${CMAKE_SOURCE_DIR}/3rdparty/opencv/opencv/build"
+    WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}/3rdparty/opencv/opencv/build"
+)
+set(INCLUDE_HEADERS "${CMAKE_CURRENT_SOURCE_DIR}/Reader_img.hpp")
+set(SRC_FILES "${CMAKE_SOURCE_DIR}/../itlab_2023/src/read/Reader_img.cpp")
+add_library(MyInclude STATIC ${INCLUDE_HEADERS} ${SRC_FILES})
```

Comment:
please use correct name for library instead `MyInclude`, `MyInclude` doesn't help to understand what happen